test/lib/tasks/buildThemes.js

```diff
@@ -5,6 +5,10 @@ const mock = require("mock-require");
 
 let buildThemes = require("../../../lib/tasks/buildThemes");
 
+test.before(() => {
+	require("@ui5/logger").setLevel("verbose");
```

Comment:
Was this intentionally added or is it a leftover from some debugging?

Reply:
Intensionally, to also cover the verbose logging code.
Should I add a comment? I actually removed it as I thought it would be clear 😄